src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -83,6 +83,14 @@ private JObject GetInvokeResult(byte[] script, Signer[] signers = null, Witness[
                     obj["state"] = ToJson(n.State, session);
                     return obj;
                 }));
+                json["logs"] = new JArray(session.Engine.Notifications.Select(n =>
```

Comment:
This is incorrect.

Reply:
This logic was when i try to add the log in the core, checking and removing useless code.